### PR TITLE
CY-720 Add 'default sort column' to models

### PR DESCRIPTION
**Pull request by tehasdf**
So that queries are ordered by something even if the user didn't
request any sorting (so that pagination makes sense).

### CODE REVIEW

File: rest-service/manager_rest/storage/models_base.py [hunk 1]
```diff
@@ -216,6 +216,15 @@ def _get_identifier_dict(self):
...
         return 'id'
 
+    @classmethod
+    def default_sort_column(cls):
+        """If no sort is requested, order by the column specified by this.
+
+        This is so that requests with pagination make sense even with no
+        sort requested by the user.
+        """
+        return getattr(cls, cls.unique_id())
```
Comment (isaac-s) [inline rest-service/manager_rest/storage/models_base.py]: Does `cls.unique_id()` translate to a DB-proprietary sequence?

Comment (tehasdf) [reply rest-service/manager_rest/storage/models_base.py]: it returns whatever the class wants to return, in this case `id` (SQLModelBase) or `_storage_id` (SQLResourceBase) (so for now, yes)

File: rest-service/manager_rest/storage/storage_manager.py [hunk 2]
```diff
@@ -83,6 +83,10 @@ def _sort_query(query, sort=None):
                 if order == 'desc':
                     column = column.desc()
                 query = query.order_by(column)
+        else:
+            default_sort = model_class.default_sort_column()
+            if default_sort:
```
Comment (mcouthon) [inline rest-service/manager_rest/storage/storage_manager.py]: How can it be empty?

Comment (tehasdf) [reply rest-service/manager_rest/storage/storage_manager.py]: the implementor of the model class might well do a `return None` if they want no default sort for that model

**Pull request merged**